packages/core/src/browser/shell/tab-bars.ts

```diff
@@ -141,11 +141,12 @@ export class TabBarRenderer extends TabBar.Renderer {
      * Render tabs with the default DOM structure, but additionally register a context menu listener.
      * @param {SideBarRenderData} data Data used to render the tab.
      * @param {boolean} isInSidePanel An optional check which determines if the tab is in the side-panel.
+     * @param {boolean} isPartOfHiddenTabBar Optional - Tells us if this entry is part of the hidden horizontal tab bar.
```

Comment:
minor: aligning the description similarly to `isInSidePanel`.

```suggestion
     * @param {boolean} isPartOfHiddenTabBar An optional check which determines if the tab is in the hidden horizontal tab bar.
```